### ISSUE TITLE
Fix github actions

### ISSUE THREAD

**Issue by DeVuDeveloper** [trkin/kindergarten-exchange#53]

**Comment by duleorlovic** [comment]
Can you put example error you are facing ?
You can put a screenshot
Please configure your IDE to use standardrb https://github.com/trkin/linters_and_code_style_guides https://github.com/testdouble/standard/wiki

**Comment by DeVuDeveloper** [comment]
I told you many times. Standard and Rubocop are not the same.  Now in GitHub actions, you don't cover rubocop offenses if you forget to do them locally. And actions will be green. Also tailwind errors you don't see in actions. It is possible to make with yml locally and take yml from the remote repo.

**Comment by duleorlovic** [comment]
> I told you many times. Standard and Rubocop are not the same. 

I agree that Standard and Rubocop are not the same (Standard uses Rubocop with it's own style configuration). We use Standard (not Rubopcop directly) with `standardrb` command

> Now in GitHub actions, you don't cover rubocop offenses if you forget to do them locally. And actions will be green. 

Can you put example for this ?

> Also tailwind errors you don't see in actions. It is possible to make with yml locally and take yml from the remote repo.

What are the tailwind errors. Please put an example

**Comment by DeVuDeveloper** [comment]
![linter-tailwind](https://user-images.githubusercontent.com/93735137/231377677-1b1f47d9-a74b-47bd-b11b-cfc0dd5f4c6b.png)

It shows errors or gives warnings for the tailwind class. Works locally and on Github actions checks.
There is no source for this.
I can give hint. Use stylelint or eslint and make custom configuration for Rails, to check only html.erb files.
If it ask for the parser, it should be added.
Rules can be added also, whatever you like.

**One more thing.** 
I am happy to meet all of you guys, but I will have busy days in the next few months, so I need to stop with practice.
Thank you for your shared knowledge and your time.

**Comment by DeVuDeveloper** [comment]
Just checked Tailwind here and a log is:
![kinder](https://user-images.githubusercontent.com/93735137/231387050-6fbf9b92-4f53-4793-818d-39c025ff9c22.png)

**Comment by duleorlovic** [comment]
@DeVuDeveloper thank you for your time.
For the screenshots I'm not sure which command you have used, so please put the command so I can try to reproduce
